cfme/tests/cloud/test_cloud_timelines.py

```diff
@@ -271,7 +274,10 @@ def test_cloud_timeline_create_event(new_instance, soft_assert, azone):
     inst_event.catch_in_timelines(soft_assert, targets)
 
 
-@pytest.mark.meta(blockers=[BZ(1542962, forced_streams=['5.8'])])
+@pytest.mark.meta(blockers=[BZ(1542962, forced_streams=['5.8']),
```

Comment:
Required:
Please remove `5.8` BZ blocker

Reply:
Most recent push removed both of these